library/Fission/Web/DNS.hs

```diff
@@ -26,10 +27,10 @@ type API
 -- Deprecated! Works the "old" way with direct access to username.fission.name,
 -- WITHOUT the `files` prefix
 server ::
-  ( MonadTime     m
-  , MonadThrow    m
-  , MonadLogger   m
-  , User.Modifier m
+  ( MonadTime       m
+  , MonadThrow      m
+  , MonadLogger     m
+  , User.Modifier   m
```

Comment:
Why?

Reply:
old formatting :unamused: 
really need to get `stylish-haskell` working